docs/reference/search/search-your-data/knn-search.asciidoc

```diff
@@ -406,6 +412,54 @@ over all documents that match the search. So for approximate kNN search, aggrega
 nearest documents. If the search also includes a `query`, then aggregations are calculated on the combined set of `knn`
 and `query` matches.
 
+[discrete]
+==== Search multiple KNN fields
```

Comment:
(Drive by, but I was interested in the change and took a look at the doc preview) 

This PR uses `KNN` but the rest of the doc uses `kNN`.  
I presume we want `kNN` everywhere.

```suggestion
==== Search multiple kNN fields
```